Sources/ProjectDescription/Settings.swift

```diff
@@ -1,22 +1,63 @@
 import Foundation
 
+// MARK: - SettingValue
+
+public enum SettingValue: ExpressibleByStringLiteral, ExpressibleByArrayLiteral, Equatable, Codable {
```

Comment:
nice use of the `ExpressibleBy` API 👍

Reply:
Should we make [this](https://github.com/tuist/tuist/blob/master/Sources/ProjectDescription/InfoPlist.swift#L4) a standalone model that can be reused from other models like `Settings`?

Reply:
@pepibumur sounds like a good idea if we have another client of that struct. I think settings won't be able to leverage it as they do not have recursive value semantic.